float32 camera1_conf

float32 camera2_conf

float32 camera3_conf

float32 camera4_conf
